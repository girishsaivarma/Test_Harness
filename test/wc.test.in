Hello world!
This is a test text.
It has three lines and 14 words.
