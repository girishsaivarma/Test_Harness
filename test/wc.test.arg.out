3 14 67 ./test/wc.test.in
